zebrad/tests/common/lightwalletd/wallet_grpc_test.rs

```diff
@@ -145,6 +143,29 @@ pub async fn run() -> Result<()> {
     // Connect to the lightwalletd instance
     let mut rpc_client = connect_to_lightwalletd(lightwalletd_rpc_port).await?;
 
+    // Check if zebrad and lightwalletd are both in the same height.
+
+    // Get the block tip from lightwalletd
+    let block_tip = rpc_client
+        .get_latest_block(ChainSpec {})
+        .await?
+        .into_inner();
+
+    // Get the block tip from zebrad
+    let client = reqwest::Client::new();
+    let res = client
+        .post(format!("http://{}", &zebra_rpc_address.to_string()))
+        .body(r#"{"jsonrpc": "2.0", "method": "getblockchaininfo", "params": [], "id":123 }"#)
+        .header("Content-Type", "application/json")
+        .send()
+        .await?
+        .text()
+        .await?;
+
+    // Make sure tip from lightwalletd and from zebrad are the same
+    let parsed: serde_json::Value = serde_json::from_str(&res)?;
+    assert_eq!(block_tip.height, parsed["result"]["blocks"]);
```

Comment:
We can move this into its own function and call it from `wait_for_zebrad_and_lightwalletd_sync`.

Here we'll want `wait_for_zebrad_tip = true`.